### PR TITLE
Parametrize version in topics

### PR DESCRIPTION
**Pull request by feymartynov**

### CODE REVIEW

File: src/mqtt.rs [hunk 1]
```diff
@@ -1239,28 +1270,32 @@ impl<'a> SubscriptionTopic for EventSubscription<'a> {
...
+                "agents/{agent_id}/api/{version}/out/{app}",
                 agent_id = from_agent_id,
+                version = ver.unwrap_or(me_version),
                 app = me.as_account_id(),
             )),
-            Source::Multicast(None) => Ok(format!(
-                "agents/+/api/v1/out/{app}",
+            Source::Multicast(None, ver) => Ok(format!(
+                "agents/+/api/{version}/out/{app}",
+                version = ver.unwrap_or(me_version),
```
Comment (manifest) [inline src/mqtt.rs]: It has to be `ver.unwrap_or("+")` (or just "+").

```
// <- request(app-from-any): agents/+/api/VER/out/ACCOUNT_ID(ME)
// <- request(app-from-any): agents/+/api/+/out/ACCOUNT_ID(ME)
```

File: src/mqtt.rs [hunk 2]
```diff
@@ -1239,28 +1270,32 @@ impl<'a> SubscriptionTopic for EventSubscription<'a> {
...
     where
         A: Addressable,
     {
         match self.source {
-            Source::Multicast(Some(ref from_agent_id)) => Ok(format!(
-                "agents/{agent_id}/api/v1/out/{app}",
+            Source::Multicast(Some(ref from_agent_id), ver) => Ok(format!(
+                "agents/{agent_id}/api/{version}/out/{app}",
                 agent_id = from_agent_id,
+                version = ver.unwrap_or(me_version),
```
Comment (manifest) [inline src/mqtt.rs]: It has to be `ver.unwrap_or("+")`.

```
// <- request(app-from-any): agents/AGENT_ID/api/+/out/ACCOUNT_ID(ME)
// <- request(app-from-any): agents/AGENT_ID/api/VER/out/ACCOUNT_ID(ME)
```

File: src/mqtt.rs [hunk 3]
```diff
@@ -131,20 +131,25 @@ impl AgentBuilder {
...
+        }
     }
 
     pub fn id(&self) -> &AgentId {
         &self.id
     }
 
     pub fn publish(&mut self, message: Box<dyn Publishable>) -> Result<(), Error> {
-        let topic = self.id.destination_topic(&message)?;
+        let topic = self.id.destination_topic(&message, &self.version)?;
```
Comment (manifest) [inline src/mqtt.rs]: Things are getting uglier. [ULMS-748](https://burning-heart.atlassian.net/browse/).

File: src/lib.rs [hunk 4]
```diff
@@ -125,27 +125,29 @@ impl FromStr for SharedGroup {
...
-    Broadcast(&'a AccountId, &'a str),
-    // <- request(app-from-any): agents/+/api/v1/out/ACCOUNT_ID(ME)
-    Multicast(Option<&'a AgentId>),
-    // <- request(one-from-one): agents/AGENT_ID(ME)/api/v1/in/ACCOUNT_ID
-    // <- request(one-from-any): agents/AGENT_ID(ME)/api/v1/in/+
-    // <- response(one-from-one): agents/AGENT_ID(ME)/api/v1/in/ACCOUNT_ID
-    // <- response(one-from-any): agents/AGENT_ID(ME)/api/v1/in/+
+    // <- event(any-from-app): apps/ACCOUNT_ID/api/VER/BROADCAST_URI
+    Broadcast(&'a AccountId, &'a str, &'a str),
+    // <- request(app-from-any): agents/+/api/+/out/ACCOUNT_ID(ME)
```
Comment (manifest) [inline src/lib.rs]: According to the code bellow: +1 option
```
// <- request(app-from-any): agents/+/api/VER/out/ACCOUNT_ID(ME)
```

File: src/mqtt.rs (hunk 3 above)
Comment (feymartynov) [reply src/mqtt.rs]: Yes. We don't have `VER(ME)` in `AgentId` so I had to pass it as an argument.

**Pull request merged**